### ISSUE TITLE
[Question]What's the different between this project and ts-opaque ?

### ISSUE THREAD

**Issue by yw662** [modfy/nominal#3]
This project seems to be a simplified version of that one, from what I read you seem to be using similar approaches to make the magic happen. I haven't read the entire code so I am probably wrong.
But anyway can the `nominal` type be used as a better/newer/shorter/lighter alternative to `opaque` type, since they are conceptually similar ?

**Comment by CryogenicPlanet** [comment]
Haven't played with `ts-opaque` much myself so not sure if they really interop, but just quickly looking at it I think `nominal` can do everything it does.

We have been adding some stuff from how we internally use `nominal` to it's "standard library" like https://github.com/modfy/nominal/blob/master/examples/proportionalityConstant.ts or things like `keys, plus, minus` here https://github.com/modfy/nominal/blob/master/src/standardLib.ts

Basically `nominal` is just something we use extensively internally and open sourced, so kinda add to it depending on what we see we need
